### ISSUE TITLE
accessibility issue in README  example svg

### ISSUE THREAD

**Issue by masukomi** [tldr-pages/tldr#8409]
![CleanShot 2022-08-27 at 09 10 55@2x](https://user-images.githubusercontent.com/2829/187032021-685761de-5b5a-44a1-9363-3809c676f90d.png)

There's a longstanding [SVG rendering bug in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1677988) when a user has a minimum font size set (folks who need larger fonts because of vision issues do this). This makes example SVGs literally unreadable. Neither Firefox, nor the tool that generated your SVG seem to be interested in fixing it, but it's fairly easy to compensate for by running the following command on your SVG.

```
sed -i '' 's/<text/<text font-size="1.67"/g' path/to/my.svg
```

To test this run firefox with font size 20 and MINIMUM font size also 20.

**Comment by CleanMachine1** [comment]
I don't even have a min of 20 on Firefox and I get this.

I'd like to see this change if it fixes it.

Would you like to submit the change in a PR @masukomi

**Comment by jxu** [comment]
I guess SVG support is still buggy in 2022.

How about just changing it to an old-fashioned GIF?

**Comment by jxu** [comment]
#5400 had a suggestion to just use a PNG. I agree in that I don't think the animation adds anything a screenshot wouldn't provide.

**Comment by jxu** [comment]
And maybe we can include just the terminal output. An actual image isn't strictly necessary but gives a visual example of output.

**Comment by fallenpixel** [comment]
Implemented on #8720

**Comment by masukomi** [comment]
Thanks so much @fallenpixel
